src/main/resources/static/views/welcome/welcome.html

```diff
@@ -5,16 +5,19 @@
             <h1 class="portalheader">
                 <i class="icon-drawer" aria-hidden="true"></i> Dashboard
             </h1>
-            <div ng-cloak ng-show="studentHasProject">
+            <div ng-cloak ng-show="!studentHasProject">
                 <h4 class="h4TileHeader"><i class="icon-home"></i> Project Information</h4>
+                <div class="alert alert-danger" ng-cloak ng-show="!isProfOrCoordinator">
+                    Your oral presentation is scheduled for <strong>Monday March 12 from 1:00PM to 2:00PM.</strong>
+                </div>
                 <div class="row">
                     <div class="col-sm-7">
                         <div class="well">
                             <h4 class="h4TileHeader"><i class="icon-notebook"></i> Project Details</h4>
                             <p><strong>Project ID:</strong> 24</p>
                             <p><strong>Title:</strong> Secure Electronic Communication Platform</p>
                             <p><strong>Supervisor:</strong> Jason Jaskolka</p>
-                            <p><strong>Second Reader:</strong> David Huang</p>
+                            <p><strong>Oral Presentation:</strong> Monday March 12 from 1:00PM to 2:00PM.</p>

```

Comment:
we don't need this anymore right?

Reply:
Ya I got rid of the second reader and added the oral presentation field instead.